core/snippets/snippets/talon.snippet

```diff
@@ -0,0 +1,9 @@
+language: talon
+---
+
+name: voiceCommandDeclaration
+phrase: command
+insertionScope: command
+-
+$0: user.vscode("$CLIPBOARD")
```

Comment:
here's an example where a no-op insertion formatter would be useful. I have that set up in my repo, so I just say "snip command blah blah" to get `blah blah: user.vscode("...")`

Reply:
Please go ahead! :)